File: Hilos Java/Ficheros texto_binario/configuracion.java
import java.io.FileNotFoundException;
import java.io.FileOutputStream;
import java.io.IOException;
import java.util.Properties;

public class configuracion {
    public static void main(String[] args) {
        Properties configuracion = new Properties();

         String miUsuario = "Admin";
         String miContrasena = "12345";
         String  elServidor = "localhost";
         String elPuerto = "8080";

         configuracion.setProperty("user", miUsuario);
         configuracion.setProperty("password", miContrasena);
         configuracion.setProperty("server", elServidor);
         configuracion.setProperty("port", elPuerto);
         try{
            configuracion.store(new FileOutputStream("configuracion.conf")
            ,"Fichero de configuracion");
         }catch(FileNotFoundException fnfe){

         }catch(IOException e){
            
         }
    }
}

File: Ficheros texto_binario/configuracion.py
import configparser

config = configparser.ConfigParser()

mi_usuario = "Admin"
mi_contrasena = "12345"
el_servidor = "localhost"
el_puerto = "8080"

config["DEFAULT"] = {
    "user": mi_usuario,
    "password": mi_contrasena,
    "server": el_servidor,
    "port": el_puerto
}

archivo_config = "configuracion.conf"

try:
    with open(archivo_config, "w") as configfile:
        config.write(configfile)
        configfile.write("# Fichero de configuración\n")
    print(f"Archivo de configuración guardado en: {archivo_config}")
except Exception as e:
    print(f"Error al guardar la configuración: {e}")
